xrmocap/utils/time_utils.py

```diff
@@ -0,0 +1,91 @@
+import time
+from xrprimer.utils.log_utils import get_logger, logging
+
+
+class Timer:
+
+    def __init__(self, name: str, logger: logging.Logger = None) -> None:
+        """Timer initialization.
+
+        Args:
+            name (str): id information of this class
+            logger (logging.Logger, optional): Save information for
+            data recording and error reporting
+            Defaults to None.
+        """
+        self.name = name
+        self.logger = get_logger(logger)
+        self.reset()
+
+    def start(self) -> None:
+        """Timer start and save current time."""
+        self.start_time = time.time()
+        self.end_time = None
+
+    def stop(self) -> None:
+        """Timer stop and save current time to calculate and save data of one
+        forward process.
+
+        Raises:
+            ValueError: start_time has not been assigned
+        """
+        if self.start_time is None:
+            self.logger.error(f'Timer {self.name} has not been started.',
```

Comment:
This branch has not been tested. See comment https://github.com/openxrlab/xrmocap/pull/90/files#r1128905314 or building action.